imap_processing/lo/packet_definitions/lo_xtce.xml

```diff
@@ -2940,7 +2940,7 @@
 			<xtce:Parameter name="ILO_SCI_CNT.SHCOARSE" parameterTypeRef="ILO_SCI_CNT.SHCOARSE" shortDescription="CCSDS Secondary Header MET">
 				<xtce:LongDescription>CCSDS Secondary Header MET</xtce:LongDescription>
 			</xtce:Parameter>
-			<xtce:Parameter name="ILO_SCI_CNT.SCI_DE" parameterTypeRef="ILO_SCI_CNT.SCI_DE" shortDescription="Binary SCI DE data" />
+			<xtce:Parameter name="ILO_SCI_CNT.SCI_CNT" parameterTypeRef="ILO_SCI_CNT.SCI_CNT" shortDescription="Binary SCI DE data" />
```

Comment:
Are you manually updating these, or did you get a new spreadsheet? Just wondering if we're going to end up overwriting this again in the future possibly and need to make a note.
```suggestion
			<xtce:Parameter name="ILO_SCI_CNT.SCI_CNT" parameterTypeRef="ILO_SCI_CNT.SCI_CNT" shortDescription="Binary SCI CNT data" />
```

Reply:
I manually updated it in the XTCE, but also updated the spreadsheet. I'll fix that missed DE instance